### PR TITLE
Use dynamically generated VC and remove expired static VCs

### PR DESCRIPTION
**Pull request by JSAssassin**
~~NOTE: Now that the VC which is being sent to various verifiers is dynamically generated using DB's VC API issuer. Anyone  running the test suite should have access to the DB client secret. Updated to use Danube Tech's issuer to generate VC for verifying.~~

Update: Made issuer configurable and it defaults to `Digital Bazaar`.

### CODE REVIEW

**Review by aljones15 — COMMENTED** [review]
I think we need to figure out which issuers to use where and why.

File: tests/10-vc-di.js [hunk 1]
```diff
@@ -22,16 +26,27 @@ describe('Verify Credential - Data Integrity', function() {
...
-  this.notImplemented = [...nonMatch.keys()];
-  for(const [verifierName, {verifiers}] of match) {
+  this.notImplemented = [...nonMatchingVerifiers.keys()];
+  for(const [verifierName, {verifiers}] of matchingVerifiers) {
     const verifier = verifiers.find(
       verifier => verifier.tags.has('VC-API'));
     describe(verifierName, function() {
+      let validVc;
+      before(async function() {
+        const issuer = matchingIssuers.get('Danube Tech').issuers.find(
```
[the remark this answers is not in the document]
Comment (JSAssassin) [reply tests/10-vc-di.js:41]: If we use our own issuer here instead, we'll have to provide all the users of the test suite with our client secret. I wasn't sure if that is something we want to do.

[the remark this answers is not in the document]
Comment (aljones15) [reply tests/10-vc-di.js:41]: Agreement here is is to make the name of the issuer used configurable.

**Review by aljones15 — APPROVED** [review]
LGTM we might want to work on the name of the environment variable in the future, but I think this is good. Good work!

**Pull request merged**